src/js/components/HeaderMenu.js

```diff
@@ -45,13 +45,13 @@ export default class Menu extends React.PureComponent {
               Index
             </Link>
           </li>
-          {/* // TODO Enable when desktop feature is ready
-          <li>
-            <Link to="/sync" onClick={toggleMenu}>
-              Sync
+          {
```

Comment:
I think @maneetpaul ji wanted to have this link shown. 

Veerji, Can you please confirm?

Reply:
Yes I was hoping we could expose on the web side now and then get the desktop side into beta / release soon.

Reply:
Personally I would wait till release sync on desktop since this a small code change for web. Ideally we should think about adding feature flags to easily control this. 

Your call though.

Reply:
Lets remove the link for now please.

Reply:
Removed the link, for now, will create a separate PR for just exposing the sync link.